### ISSUE TITLE
fundrawtransaction address re-use

### ISSUE THREAD

**Issue by gituser** [dashpay/dash#1749]
Hi.

It seems latest `Dash Core Daemon version v0.12.2.1` have this bug https://github.com/bitcoin/bitcoin/issues/9362

Could you guys backport the fix and add an option to fundrawtransaction to specify a `changeAddress` and `feeRate` as well?
from bitcoin-cli help fundrawtransaction
```
2. options                 (object, optional)
   {
     "changeAddress"          (string, optional, default pool address) The bitcoin address to receive the change
     ....
     "feeRate"                (numeric, optional, default not set: makes wallet determine the fee) Set a specific feerate (BTC per KB)
    ....
   }
```

**Comment by codablock** [comment]
We have a huge list of PRs to backport and the fix for this one is part of that list. It will however take some time till we release a version with all the PRs backported.

**Comment by colinrmoody** [comment]
@codablock Any update on this issue? If you guys could add option to specify a changeAddress that would be very helpful. I'm trying to create/fund transactions with a watch-only wallet so that I can sign offline in cold storage, but without this feature all my change goes to a newly created address on the hot machine (separate from the offline wallet). The only fix is to create another transaction that sends the change back.

**Comment by UdjinM6** [comment]
@colinrmoody The fix has been backported and is a part of 0.12.3.x branch now:
>36109e058 Merge #9377: fundrawtransaction: Keep change-output keys by default, make it optional

https://github.com/dashpay/dash/blob/master/doc/release-notes/dash/release-notes-0.12.3-backports.md

Pls try 0.12.3.2 release.

**Comment by UdjinM6** [comment]
Should be fixed. Closing.
